modules/mono/editor/GodotTools/GodotTools/Export/AotBuilder.cs

```diff
@@ -547,9 +547,9 @@ private static IEnumerable<string> GetEnabledAndroidAbis(string[] features)
         {
             var androidAbis = new[]
             {
-                "armeabi-v7a",
-                "arm64-v8a",
-                "x86",
+                "arm32",
+                "arm64",
+                "x86_32",
```

Comment:
This is probably wrong. It doesn't match the changes you made in `mono_configure.py`, as you can see those were the values from the dict, not the keys.